src/sqlancer/cockroachdb/CockroachDBProvider.java

```diff
@@ -273,10 +307,14 @@ public SQLConnection createDatabase(CockroachDBGlobalState globalState) throws S
             port = CockroachDBOptions.DEFAULT_PORT;
         }
         String databaseName = globalState.getDatabaseName();
-        String url = String.format("jdbc:postgresql://%s:%d/test", host, port);
+        String url = buildPGUrl(host, port, databaseName);
+        System.out.println("JDBC url="+ url);
```

Comment:
It would be great if we could remove such print statements before merging the change.